src/app/components/Text/index.jsx

```diff
@@ -13,6 +13,24 @@ const StyledParagraph = styled.p`
     line-height: 1.375rem;
   }
 `;
+const StyledItalic = styled.i`
```

Comment:
We don't need to set the styles of this element, they will always be wrapped in a paragraph and so will inherit the paragraph's styles. This should just be a simple React component that returns `i` with `children` passed to it.